### PR TITLE
[API] [ExchangeRate] CRUD

### PR DESCRIPTION
**Pull request by lchrusciel**
| Q               | A
| --------------- | ---
| Bug fix?        | no |
| New feature?    | yes |
| BC breaks?      | yes |
| Related tickets |  |
| License         | MIT |

### CODE REVIEW

File: src/Sylius/Bundle/CurrencyBundle/Validator/Constraints/UniqueCurrencyPairValidator.php [hunk 1]
```diff
@@ -41,6 +41,10 @@ public function __construct(ExchangeRateRepositoryInterface $exchangeRateReposit
      */
     public function validate($value, Constraint $constraint)
     {
+        if (null !== $value){
```
Comment (GSadee) [inline src/Sylius/Bundle/CurrencyBundle/Validator/Constraints/UniqueCurrencyPairValidator.php]: Space before `{`

File: src/Sylius/Bundle/CurrencyBundle/Resources/config/serializer/Model.ExchangeRate.yml [hunk 2]
```diff
@@ -9,8 +9,11 @@ Sylius\Component\Currency\Model\ExchangeRate:
         ratio:
             expose: true
             type: float
+            groups: [Default, Detailed]
         sourceCurrency:
             expose: true
+            groups: [Default, Detailed]
         targetCurrency:
             expose: true
+            groups: [Default, Detailed]
```
Comment (michalmarcinkowski) [inline src/Sylius/Bundle/CurrencyBundle/Resources/config/serializer/Model.ExchangeRate.yml]: Having `updatedAt` field on `ExchangeRate` model could be useful, but that's something that can be added later as well 😉

File: src/Sylius/Bundle/CurrencyBundle/Validator/Constraints/UniqueCurrencyPairValidator.php [hunk 3]
```diff
@@ -41,6 +41,10 @@ public function __construct(ExchangeRateRepositoryInterface $exchangeRateReposit
...
+            return;
```
Comment (michalmarcinkowski) [inline src/Sylius/Bundle/CurrencyBundle/Validator/Constraints/UniqueCurrencyPairValidator.php]: Seems wrong, if the value is not null, then do nothing...

**Comment by pjedrzejewski** [comment]
Will be merged in #7543.

**Pull request closed without merging**